db/migrate/20230905150033_create_recipes.rb

```diff
@@ -0,0 +1,14 @@
+class CreateRecipes < ActiveRecord::Migration[7.0]
+  def change
+    create_table :recipes do |t|
+      t.string :name
+      t.string :preparation_time
+      t.string :cooking_time
```

Comment:
- [ ] I think the preparation and cooking time should be defined as time or float if you want and not string 🙃 , kindly fix that 🙏🏾